test/test.js

```diff
@@ -0,0 +1,52 @@
+const {createS3UploadParams, reverse} = require("../index");
+const assert = require("assert");
+
+describe("Reverse", function () {
+  const tests = [
+    { word: "abc", expected: "cba" },
+    { word: "abc ", expected: "cba" }, // removes trailing spaces
+    { word: " abc", expected: "cba" }, // removes leading spaces
+    { word: "ab c", expected: "c ba" }, // keeps spaces between non-space characters
+  ];
+
+  tests.forEach(({ word, expected }) => {
+    it(`reverses ${word} correctly`, function () {
+      assert.equal(reverse(word), expected);
+    });
+  });
+});
+
+describe("CreateS3UploadParams", function () {
+  const ip_ranges_example = {
+    syncToken: "0123456789",
+    createDate: "yyyy-mm-dd-hh-mm-ss",
+    prefixes: [
+      {
+        ip_prefix: "cidr",
+        region: "region",
+        network_border_group: "network_border_group",
+        service: "subset",
+      }
```

Comment:
Codacy has a fix for the issue: [Replace `············}` with `······},`](https://app.codacy.com/gh/dmgress/aws-ip-ranges/pullRequest?prid=9261990)
```suggestion
      },
```